keyboards/nightly_boards/alter/rev1/info.json

```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "Alter",
+    "url": "",
+    "maintainer": "Neil Brian Ramirez",
+    "width": 20,
+    "height": 5.25,
+    "layouts": {
+        "LAYOUT": {
+            "layout": [{"x":0.5, "y":0.15}, {"x":1.75, "y":0.24999999999999997}, {"x":2.75, "y":0.24999999999999997}, {"x":3.75, "y":0.24999999999999997}, {"x":4.75, "y":0.24999999999999997}, {"x":5.75, "y":0.24999999999999997}, {"x":6.75, "y":0.24999999999999997}, {"x":7.75, "y":0.24999999999999997}, {"x":11.25, "y":0.24999999999999997}, {"x":12.25, "y":0.24999999999999997}, {"x":13.25, "y":0.24999999999999997}, {"x":14.25, "y":0.24999999999999997}, {"x":15.25, "y":0.24999999999999997}, {"x":16.25, "y":0.24999999999999997}, {"x":17.25, "y":0.24999999999999997}, {"x":18.25, "y":0.24999999999999997}, {"x":0.25, "y":1.15}, {"x":1.5, "y":1.25, "w":1.5}, {"x":3, "y":1.25}, {"x":4.25, "y":1.25}, {"x":5.25, "y":1.25}, {"x":6.25, "y":1.25}, {"x":7.25, "y":1.25}, {"x":10.75, "y":1.25}, {"x":11.75, "y":1.25}, {"x":12.75, "y":1.25}, {"x":13.75, "y":1.25}, {"x":15, "y":1.25}, {"x":16, "y":1.25}, {"x":17, "y":1.25}, {"x":18, "y":1.25, "w":1.5}, {"x":0, "y":2.15}, {"x":1.25, "y":2.25, "w":1.75}, {"x":3, "y":2.25}, {"x":4.5, "y":2.25}, {"x":5.5, "y":2.25}, {"x":6.5, "y":2.25}, {"x":7.5, "y":2.25}, {"x":11, "y":2.25}, {"x":12, "y":2.25}, {"x":13, "y":2.25}, {"x":14, "y":2.25}, {"x":15.5, "y":2.25}, {"x":16.5, "y":2.25}, {"x":17.5, "y":2.25, "w":2.25}, {"x":1, "y":3.25, "w":2.25}, {"x":3.25, "y":3.25}, {"x":5, "y":3.25}, {"x":6, "y":3.25}, {"x":7, "y":3.25}, {"x":8, "y":3.25}, {"x":10.5, "y":3.25}, {"x":11.5, "y":3.25}, {"x":12.5, "y":3.25}, {"x":13.5, "y":3.25}, {"x":15.25, "y":3.25}, {"x":16.25, "y":3.25}, {"x":17.25, "y":3.25, "w":1.75}, {"x":19, "y":3.25}, {"x":1, "y":4.25, "w":1.5}, {"x":4.75, "y":4.25, "w":1.5}, {"x":6.25, "y":4.25, "w":2}, {"x":8.25, "y":4.25, "w":1.25}, {"x":10.5, "y":4.25, "w":2.75}, {"x":13.25, "y":4.25, "w":1.5}, {"x":18.25, "y":4.25, "w":1.5}]
```

Comment:
This seems to have some rounding errors, probably because you had rotated keys and the KLE-to-info.json converter doesn't support that - could you please remove the rotation and regenerate this file?

Reply:
I updated it on my latest commit.

Reply:
https://github.com/qmk/qmk_firmware/pull/9579/commits/28b64ad32f645eb0f4b44b3a95ff578a32a63386